Makefile

```diff
@@ -40,6 +40,12 @@ all: copr-rpms rpms
 local:
 	$(MAKE) RPM_DIST="0.$(shell date '+%s')" all
 
+check:
```

Comment:
Technically, blank and rustfmt are changing the source code, so  "check" is misleading.

Reply:
`fmt` would be a better target name

Reply:
Though there are things for both formatting and checking. Perhaps create two targets `fmt` and `check`, where `check` depends on `fmt`